cns/restserver/restserver.go

```diff
@@ -281,3 +306,19 @@ func (service *HTTPRestService) Stop() {
 	service.Uninitialize()
 	logger.Printf("[Azure CNS]  Service stopped.")
 }
+
+// MustGenerateCNIConflistOnce will generate the CNI conflist once if the service was initialized with
+// a conflist generator. If not, this is a no-op.
+func (service *HTTPRestService) MustGenerateCNIConflistOnce() {
+	service.generateCNIConflistOnce.Do(func() {
+		if err := service.cniConflistGenerator.Generate(); err != nil {
+			logger.Errorf("unable to generate cni conflist with error: %v", err)
+			os.Exit(1)
```

Comment:
Not crazy about `os.Exit` appearing anywhere other than `main()` because it prevents `defers` from running elsewhere. Can we return here instead? Alternatively, `runtime.Goexit()` is slightly better, since it's like a goroutine-specific `os.Exit` (it's how `testing.(*T).Fatal` works under the hood).

It's a bit tricky to see the forest through the trees on the diff though, so if it's a pervasive thing, don't let it block the PR.

Reply:
changed to `panic`